forage-android/src/main/res/values/attrs.xml

```diff
@@ -12,6 +12,16 @@
         <attr name="android:textColor" />
         <!-- Reusing android:textColor attribute -->
         <attr name="android:textSize" />
+
+        <attr name="cornerRadius" />
+        <attr format="dimension" name="boxCornerTopEnd" />
+        <attr format="dimension" name="boxCornerTopStart" />
+        <attr format="dimension" name="boxCornerBottomStart" />
+        <attr format="dimension" name="boxCornerBottomEnd" />
+
+        <attr name="android:strokeColor" />
+        <attr name="android:strokeWidth" />
+        <attr name="boxStrokeWidthFocused" />
```

Comment:
The `boxCorner*`s all of `format=`s fields. Any particular reason we don't want the `format=` on the other `<attr />`s we're introducing in this PR?

Reply:
I added "format" fields to have consistency. Thanks!